### PR TITLE
contact form section for mobile view portfolio project

### PR DESCRIPTION
**Pull request by okemboi**
I added html for contact-form with below sections

- Full name 
- Email 
- Message box
- Form spree to send collected data after submitting the form

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @okemboi & @rabayarabu 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: 

You have submitted a project, that have be done in pair programing. I can see your partner's commits but I can not identify any commits of Mr @okemboi and as you know in order to be sure that everyone has contributed to the projects, we mus see every partner's commits. Kindly contribute to your project and submit it again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -197,6 +197,46 @@ <h3 class="frameworks">Skills</h3>
...
+        <form action="https://formspree.io/f/mvonwygw" method="POST">
+            <input type="text" name="full_name" maxlength="30" placeholder="Full name" required/>
+            <input type="email" name="email" placeholder="Email" required/>
+            <textarea name="textarea" maxlength="500" placeholder="Write me something..." required></textarea>
+            <input type="button" title="Get in touch" placeholder="Get in touch" />
+        </form>
+        <ul class="social_link">
+            <li></li>
+        </ul>
+    </section>
```
Comment (frank1738) [inline index.html:213]: - Your contact form is not styled at all. Kindly style it to match the original design.

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -197,6 +197,49 @@ <h3 class="frameworks">Skills</h3>
...
+                </li>
+            </ul>
+        </div>
+        </footer>
+
+        <div class="shape">
+            <img src="/images/Shape.png" alt="">
+        </div>
+        </div>
+    </section>
```
Comment (Qoosim) [inline index.html:242]: - [ ] Great job implementing the changes required by the previous reviewer. However, there're few things you need to improve on this milestone. It is required to stick to the figma design as close as possbile. This includes font, color, spacing among others. Your design doesn't yet look like that of figma. Kindly give a closer look and try to produce a replica of figma design.

- [ ] Also, at this moment, all the footer icons are not showing. Kindly attend to that too.

### Your Design
your-form

### Figma Design
figma-form

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @okemboi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @okemboi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**